Client/Assets/CC_Script/CreditCardExtras.shared.mjs

```diff
@@ -0,0 +1,30 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+class CreditCardExtras {
+  focusNextInputField() {
+    var inputFields = document.getElementsByTagName('input');
+    var activeElement = document.activeElement;
+    var currentIndex = Array.prototype.indexOf.call(inputFields, activeElement);
```

Comment:
Then we can call **indexOf** directly instead of acessing the prototype
```suggestion
    const currentIndex = inputFields.indexOf(activeElement);
```